spec/controllers/rents_controller_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+
+describe RentsController do
```

Comment:
Remember what we spoke about `def`s in spec files! Please, refactor this to remove them and follow [betterspecs](http://www.betterspecs.org/) suggestions!